get/redis_test.go

```diff
@@ -0,0 +1,120 @@
+package get
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"testing"
+
+	storage "github.com/ninech/apis/storage/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/internal/test"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func Test_redisCmd_Run(t *testing.T) {
```

Comment:
Any particular reason this wasn't named `TestRedis` like in "create", "delete", etc ?

Reply:
Just the library that generated the name. But I agree we should have consistent naming.